e2e/cypress/integration/accessibility/accessibility_input_fields_spec.js

```diff
@@ -50,17 +56,22 @@ function verifyMessageAutocomplete(index, type = 'user') {
 describe('Verify Accessibility Support in different input fields', () => {
     let testChannel;
 
-    before(() => {
-        // # Login as sysadmin and update Config
+    beforeEach(() => {
```

Comment:
This is already using beforeEach in channel creation. I just flattened the logic to make it simpler.